index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <header class="main-nav">
+            <div class="main-logo"><a href="#">Oscar Byiringiro</a></div>
+            <nav>   
+                <div class="bottom-icons">
+                    <img src="./images/Union.png">
+                        
+                </div>
+            </nav>
+        </header>
```

Comment:
- [ ] Good job implementing the changes suggested by the previous reviewer, but there are some things remaining to match the Figma design. 🤝️
Kindly check the toolbar size, it seems a bit bigger when compared to the template. 🥸️
Please check if you are using the correct `font-size`, every text seems small in your design. 😎️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/212827069-f51d0715-fd50-43b7-9a55-9a99357bb3c9.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/212827080-e0d2fb2b-382a-40b4-a037-a8a341da721b.png" width="300">|